setup.cfg

```diff
@@ -9,6 +9,7 @@ branch = True
 
 [coverage:report]
 show_missing = True
+fail_under = 70
```

Comment:
This, on the other hand [1], we definitely have to get higher. :) 

[1] https://github.com/aws/aws-encryption-sdk-python/pull/325/files#r602435534